std/http/file_server.ts

```diff
@@ -142,6 +142,8 @@ async function serveDir(
 }
 
 async function serveFallback(req: ServerRequest, e: Error): Promise<Response> {
+  console.error(e.message);
```

Comment:
I would have put this before the _call_ of `serveFallback()`:https://github.com/denoland/deno/blob/a66f1ed2b543eda2496f46924eb3446e386e6af1/std/http/file_server.ts#L302

Reply:
agree